azkaban-exec-server/src/main/java/azkaban/execapp/ExecutorServlet.java

```diff
@@ -368,6 +371,20 @@ private void setActive(boolean value, Map<String, Object> respMap)
     }
   }
 
+  private void getExecutorId(Map<String, Object> respMap)
+      throws ServletException {
+    try {
+      ExecutorLoader executorLoader = application.getExecutorLoader();
+      Executor executor = executorLoader.fetchExecutor(application.getHost(), application.getPort());
```

Comment:
There is a requireNonNull API in java Objects class itself that you can directly use.